troposphere/static/theme/appTheme.js

```diff
@@ -0,0 +1,7 @@
+import cyverseTheme from 'cyverse-ui/styles/cyverseTheme.json';
+import theme from './theme.json';
+import _ from 'lodash';
+
+const appTheme = _.merge(cyverseTheme, theme);
```

Comment:
I understand the semantics of `merge` for lodash are different than what is available in `underscore`, but I think we want to consider adding lodash just for `merge`.

I'm 3/4 through the 120+ files changes, and I only see this as a valid need.